packages/wdio-firefox-profile-service/tests/launcher.test.js

```diff
@@ -4,27 +4,26 @@ import FirefoxProfile from 'firefox-profile'
 describe('Firefox profile service', () => {
     describe('onPrepare', () => {
         test('should return when no firefoxProfile set in the config', async () => {
-            const config = {}
+            const options = {}
             const capabilities = [{}]
+            const config = {}
 
-            const service = new Launcher()
-            service.onPrepare(config, capabilities)
+            const service = new Launcher(options, capabilities, config)
```

Comment:
According to the changes in the service, `FirefoxProfileLauncher` constructor only takes one parameter.